jerry-core/jcontext/jcontext.h

```diff
@@ -187,6 +190,36 @@ extern jerry_hash_table_t jerry_global_hash_table;
 
 #endif /* !CONFIG_ECMA_LCACHE_DISABLE */
 
+#else /* JERRY_ENABLE_EXTERNAL_CONTEXT */
+
+#define JERRY_CONTEXT(field) (((jerry_context_t *) jerry_port_get_current_instance()->context_p)->field)
```

Comment:
Hmm, this macro is used a ton, so I think we should optimize this for code size.
I would add a separate `jerry_port_get_current_context()`, to get rid of the extra dereferencing / level of indirection.

@zherczeg what do you think?